packages/nocodb/src/models/Integration.ts

```
@@ -490,19 +511,23 @@ export default class Integration implements IntegrationType {
-      .where((whereQb) => {
+      }).where((whereQb) => {
         whereQb
           .where(`${MetaTable.PROJECT}.deleted`, false)
           .orWhereNull(`${MetaTable.PROJECT}.deleted`);
       });
+    }
+
+    const sources = await qb;
 
-    return (this.sources = sources);
+    return (this.sources = sources.map((src) => new Source(src)));
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)